### ISSUE TITLE
app folder support?

### ISSUE THREAD

**Issue by hanrw** [imPrathamDev/nextjs-ecommerce#1]
Do you have any plan to migrate to using app folder?

**Comment by imPrathamDev** [comment]
Yeah I will migrate this project to app directory and create a separate backend because I am also creating React Native app for this project

**Comment by hanrw** [comment]
> Yeah I will migrate this project to app directory and create a separate backend because I am also creating React Native app for this project

The UI looks fantastic. And it's would perfect if it's with a separate backend as well. Looking forward~
